includes/abstracts/actions/edit-shipping.php

```diff
@@ -80,7 +88,7 @@ protected function get_shipping_method_titles() {
 		$shipping_method_titles = [];
 
 		if ( $this->include_all_shipping_method ) {
-			$shipping_method_titles['all'] = __( 'All shipping methods', 'automatewoo-subscriptions' );
+			$shipping_method_titles[$this->all_shipping_method_option_key] = __( 'All shipping methods', 'automatewoo-subscriptions' );
```

Comment:
Minor: This needs spaces around variable in the array access.

```suggestion
			$shipping_method_titles[ $this->all_shipping_method_option_key ] = __( 'All shipping methods', 'automatewoo-subscriptions' );
```